.github/workflows/build.yml

```
@@ -0,0 +1,57 @@
+      if: runner.os == 'Windows'
+      run: |
+        Compress-Archive -Path dist/gmail-cleaner -DestinationPath gmail-cleaner-${{ matrix.os }}.zip
+
+    - name: Upload Artifacts
+      uses: actions/upload-artifact@v4
+      with:
+        name: gmail-cleaner-${{ matrix.os }}
+        path: gmail-cleaner-${{ matrix.os }}.zip
+
+    - name: Release
+      uses: softprops/action-gh-release@v1
```

actionlint (1.7.10)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)